### ISSUE TITLE
tf2 crashes when I rename my class files to have a "_c"

### ISSUE THREAD

**Issue by shapetf2** [mastercoms/tf2cfg#37]
## Reporting an issue (related to class configs)
The issue which I am currently facing is that when I rename my class files (e.g. scout.cfg) to (scout_c.cfg), tf2 immediately crashes and when I attempt to execute the class file (e.g. exec scout) in console, the commands are unknown and don't work. 

The version of comfig I am using is 6.5.0. The preset I am using is max performance and I am not using any addons. 
I do not believe that I am using any custom values (I just use the comfig file's premade settings and remove the double slashes on the options I want to use in-game). 
The dxlevel I am using is dx9. 
The launch options that I have are -nops2b -w 1280 -h 720 -novid -sw -nojoy -nosteamcontroller -noff -nohltv -softparticlesdefaultoff -reuse -primarysound -NoQueuedPacketThread -snoforceformat -nostartups -softparticlesdefaultoff. 
The operating system I am using is Windows 10 Home Edition version  10.0.16299 build 16299. I am using an Acer Aspire E 15 laptop with core i3-5015U cpu with integrated graphics and 8 gb of ddr3 ram. (idk how the stuff i typed out above would help me with my problem but I did it anyways :])

**Comment by mastercoms** [comment]
Remove exec autoexec from your class config.

**Comment by mastercoms** [comment]
Also, I wouldn't recommend removing double slashes, instead, paste the options you want into a custom.cfg file.

**Comment by shapetf2** [comment]
Thanks for the reply! I'll try what you suggested and see if it gets fixed.

**Comment by shapetf2** [comment]
It didn't work, my game still crashes when I exec a class file with a "_c" in the file name (e.g. scout_c.cfg). When I execute the file in the console I type "exec scout_c" and my game starts to hang a bit before it closes immediately. Also I made a custom.cfg file and pasted the options I wanted inside the file.

**Comment by shapetf2** [comment]
Alright I fixed it but am not too sure how. The fix might have happened when I deleted the alias commands that came with the comfig file but idk :/

I also made a custom.cfg and followed some of what you said (I didn't have the command "exec autoexec" in my class files anyways). Thanks for helping me! :>

**Comment by mastercoms** [comment]
Hm, could you please share your class configs?

**Comment by shapetf2** [comment]
I also think my cfg files work now because I added these commands to my class cfg files even though they weren't renamed to have a _c. Evidence of this is because when I switch to a class (e.g. sniper) my game crashes without these commands.

**these commands**
mem_compact // Occassionally organize memory in-game

exec game_overrides // Override values set every game

exec demoman_c (I used the respective classes such as engineer_c if i was using this command for engineer class cfg).
